app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+
+  belongs_to :author, class_name: 'User'
+
+  after_save :update_post_user_counter
+
```

Comment:
- [ ] you don't have a method name **update_post_user_counter** but you have a method named **post_counter** so you can put it here :smile: as shown below :+1:

`after_save :post_counter`